### DISCUSSION TITLE
[Feature Request]: Front End form for delegating meeting data entry to end user.

### DISCUSSION THREAD

**Discussion by pruggear** [code4recovery/12-step-meeting-list#714] [Q&A] [answered]
### Contact Details

pruggear@comcast.net

### Requested Feature/Enhancment

Please add this cool feature!

### TSML Version

Latest (default)

### Wordpress Version

Latest (default)

**Comment by kiyote33** [comment, marked as answer]
Hi Pamela, @pruggear 

Sorry for the slow response to this issue. The ability to delegate meeting data entry to an end user is inherent in your WordPress website, simply by setting the permissions for a specified user to the role of an editor, which allows database write permissions to users in that role on some of the available website features including the 12 Step Meeting List. 

Please note: For our work flow, we’re asking that user issues be opened inside of Discussions first. The maintainers will then migrate the details to an Issue when appropriate. I will convert this issue to a discussion topic for posterity, Thanks!

**Reply by pruggear** [reply]
Thank you very much for the response.  I will remember to go to the discussion list first! :-)

**Reply by charlesrbasin** [reply]
Users with role of editor do not have access to the meetings Import csv file. We would like to remove this burden from the admin and allow the editor person to add and update meetings in bulk using the csv file.

**Reply by kiyote33** [reply]
Ah... You would need to be in the Administrator role to be able to import all the meetings via the CSV Import feature.
